src/utils/address.ts

```diff
@@ -0,0 +1,31 @@
+import { NetworkVMType } from '@avalabs/vm-module-types';
+
+import { Account } from '@src/background/services/accounts/models';
+import { omitUndefined } from '@src/utils/object';
+
+export const mapVMAddresses = (addresses: Record<NetworkVMType, string>) =>
+  omitUndefined({
+    addressC: addresses[NetworkVMType.EVM],
+    addressBTC: addresses[NetworkVMType.BITCOIN] || undefined,
```

Comment:
If for given VM type there is no address, it would be automatically `undefined` wouldn't it? 🤔 
\#justAsking

Reply:
Technically could be an empty string

Reply:
Ah gotcha! Thanks!